### ISSUE TITLE
Add indicator in minibuffer when using a register like vim

### ISSUE THREAD

**Issue by nullbyto** [emacs-evil/evil#1755]
# Issue type

- Question/Enhancement request

# Issue
It would be nice to display the register (not the content of it), that is currently used when start typing `"+` or simply by starting to type `"` for example at the bottom (right) like in vim.
Like an indicator when typing, to get feedback from evil when starting to type `"`. It could be on the left side as well, like `evil-snipe` when outputting `1>`.
It would be really good as well if it could also append numbers or operations like `y`, `d`.

I tried to make something by my own which is kinda hacky but not exactly elegant (i don't know much e-lisp):
```elisp
(defun print-register (reg)
  (message (format "\"%c" reg))
  )
(advice-add 'evil-use-register :before 'print-register)
```
It displays only after typing the actual register like `"+`, nothing happens when typing `"`.

I would like to see what your opinion on that. Thank you for your time.

**Comment by axelf4** [comment]
This is a special case of #1386.

**Comment by nullbyto** [comment]
Indeed. What do you think about that and/or is there any plans on adding that ? Or workarounds until added, is good as well.

**Comment by axelf4** [comment]
Please try the patch in the referenced pull request.

**Comment by nullbyto** [comment]
For info: i use `doom emacs`, i have imported the commit from your fork. Tried it out, works great. 
Two notes:

- If i keep pressing double quote, this happens:
![image](https://user-images.githubusercontent.com/67614151/213819747-b4d67047-f321-4574-8ed0-0504fddd3bb2.png)
- Another thing after choosing a register like `"+` and then typing the number of times an operation should be executed, it doesn't show the numbers. Not a big deal tho

Thanks and great work!

**Comment by nullbyto** [comment]
Regarding the double quotes `" " " " ... ` it happens in vim as well but i have no idea why is that.

**Comment by winkelnp** [comment]
Any update on this? would be really great to have?
